build.gradle.kts

```diff
@@ -151,7 +151,7 @@ tasks {
                             }
                         ]
                     }
-                """.trimIndent()
+                """.trimIndent().replace("\n", "\r\n")
```

Comment:
Kotlin insists on using Unix line endings. Since we both work on Windows, this results in the auto-generated text files having different endings than rest of the project and git complaining on having to correct them.

With the `.replace("\n", "\r\n")` call the files are generated with proper CRLF line endings.